labelbox/data/cloud/blobstorage.py

```diff
@@ -0,0 +1,31 @@
+import os
+import re
+
+from azure.storage.blob import BlobServiceClient, ContainerClient
+from loguru import logger
+
+
+def extract_file_path(path: str, container_name: str) -> str:
+    """Given a blobstorage url for a file, extract the file path after
+        container name.
+        Example:
+            https:/{storage-account-name}.blob.core.windows.net/{container-name}/images/1001.jpg
+        returns
+            images/1001.jpg
+    """
+
+    file_path_regex = re.compile(f"{container_name}/(.*)")
+    return file_path_regex.search(path).group(1)
```

Comment:
I would not expect to have to pass a `container_name` in order to get the path part of a URL. _If_ we can assume that the first part of the path is always the container and that we don't want that part, how about...

```python
from urllib.parse import urlparse

def extract_file_path(path: str) -> str:
    """Extract the file path after from a blob storage URL.

    Container name (the first path segment) is excluded.

    >>> path = "https://storage-acct-name.blob.core.windows.net/container/imgs/1001.jpg"
    >>> extract_file_path(path)
    ... "imgs/1001.jpg"
    """
    return urlparse(path).path.split('/', maxsplit=2)[-1]
```